addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -5977,6 +5977,12 @@ registry.ReplaceMedia = SnippetOptionWidget.extend({
         this._deactivateLinkTool = this._deactivateLinkTool.bind(this);
     },
 
+    destroy: function () {
+        this.options.wysiwyg.odooEditor.removeEventListener('activate_image_link_tool', this._activateLinkTool);
+        this.options.wysiwyg.odooEditor.removeEventListener('deactivate_image_link_tool', this._deactivateLinkTool);
+        return this._super(...arguments);
+    },
```

Comment:
Mmh, this should probably be automatic (onBlur being called first when destroying)... but maybe not. It's ok to fix like this in stable. But at least share the code instead of copy-pasting it please :pray: Create a private function, called both in onBlur and here :pray:

Reply:
hello @qsm-odoo , thanks for the review, commit update.